User Portal Redesign - Login

lizsurette opened the pull request:
Let's use this PR to review purely the Login page redesign for the User Portal:
https://github.com/lizsurette/ovirt-design/blob/fed0fe9b1dd518a2f8b7407eb3faf49032158b38/user-portal/login.md

lizsurette committed:
User Portal Redesign - Login Page

This PR shows an initial proposal for the Login Page for the User Portal. This would make use of the new PatternFly Login: https://github.com/patternfly/patternfly-react/pull/362

gregsheremeta commented:
(@mykaul:) Login page:

Missing some hyperlinks (to the certificate, client resources, documentation?, etc.)
What is the purpose of the 'forgot password?' link? And 'sign up' . We don't have either in oVirt.
You are not logging in to your 'account' really, but to the VM portal.

lizsurette commented:
@mykaul @gregsheremeta 

> Missing some hyperlinks (to the certificate, client resources, documentation?, etc.)

Currently this is on the landing page for this information and I'd expect it to all remain there. This is after the user has selected to specifically log in to the User Portal. Would you like to see any changes here?

> What is the purpose of the 'forgot password?' link? And 'sign up' . We don't have either in oVirt.

Removed these in the latest design.

> You are not logging in to your 'account' really, but to the VM portal.

I've updated the text to read "Log In to Your Portal Account". Thoughts?

lizsurette commented:
Feedback from Moran:

1. Sign up link still needs to go.
2. We should consider a better term for 'profile'. Domain? Something else?

gregsheremeta commented:
>  better term for 'profile'

@serenamarie125 ?

sjd78 commented:
> We should consider a better term for 'profile'. Domain? Something else?

Why not just 'Log In' and leave the 'to your we-don't-have-a-good-name-for-it-yet' completely off?

lizsurette committed:
Updating image

lizsurette committed:
Updating link to where this design would live on the design documentation website.
